drivers/ethernet/Kconfig.native_posix

```diff
@@ -12,6 +12,30 @@ menuconfig ETH_NATIVE_POSIX
 	  a process in your host system.
 
 if ETH_NATIVE_POSIX
+config ETH_NATIVE_POSIX_STARTUP_AUTOMATIC
+	bool "Start network interface automatically"
+	help
+	  If set, then the native_posix ethernet driver tries to setup network
```

Comment:
Seems like too much information here, and overlaps with eth_native_posix/README.rst.
How about:

    If set, the native_posix ethernet driver will set up the network interface,
    requiring ``zephyr.exe`` to be run with root privileges (needed to create and
    configure the TAP device).  

    If not set (the default and recommended way),
    the network interface must be set up manually using ``net-setup.sh`` (from
    the net-tools project repo). The ``zephyr.exe`` program can then be run
    as a non-root user. 

And move the other details here over to the eth_native_posix/README.rst doc.